examples/realworld/main.wasp

```diff
@@ -7,29 +7,181 @@ page Main {
+    createdAt       DateTime @default(now())
+    updatedAt       DateTime @updatedAt
+    title           String
+    description     String
+    markdownContent String
+
+    user            User     @relation(fields: [userId], references: [id])
+    userId          Int
+    comments        Comment[]
+    tags            ArticleTag[]
+    favoritedBy     User[]   @relation("FavoritedArticles")
+psl=}
+
+entity Comment {=psl
+    id              Int      @id @default(autoincrement())
+    createdAt       DateTime @default(now())
+    content         String
+
+    user            User     @relation(fields: [userId], references: [id])
+    userId          Int
+    article         Article  @relation(fields: [articleId], references: [id])
+    articleId       Int
+psl=}
+
+entity ArticleTag {=psl
+    name          String    @id
+
+    articles      Article[]
 psl=}
 
 auth {
   userEntity: User,
   methods: [ EmailAndPassword ]
 }
 
+query getUser {
+  fn: import { getUser } from "@ext/user/queries.js",
+  entities: [User]
+}
+
 action signup {
-  fn: import { signup } from "@ext/actions.js",
+  fn: import { signup } from "@ext/user/actions.js",
   entities: [User]
-}
\ No newline at end of file
+}
+
+action updateUser {
+  fn: import { updateUser } from "@ext/user/actions.js",
+  entities: [User]
+}
+
+action followUser {
+  fn: import { followUser } from "@ext/user/actions.js",
+  entities: [User]
+}
+
+query getArticlesByUser {
+  fn: import { getArticlesByUser } from "@ext/article/queries.js",
+  entities: [Article]
+}
+
+query getFavoritedArticles {
+  fn: import { getFavoritedArticles } from "@ext/article/queries.js",
+  entities: [Article]
+}
+
+query getFollowedArticles {
+  fn: import { getFollowedArticles } from "@ext/article/queries.js",
+  entities: [Article, User]
+}
+
+query getAllArticles {
+  fn: import { getAllArticles } from "@ext/article/queries.js",
+  entities: [Article]
+}
+
+query getArticle {
+  fn: import { getArticle } from "@ext/article/queries.js",
+  entities: [Article]
+}
+
+query getArticleComments {
+  fn: import { getArticleComments } from "@ext/article/queries.js",
+  entities: [Comment]
+}
+
+action createArticle {
+  fn: import { createArticle } from "@ext/article/actions.js",
+  entities: [Article]
+}
+
+action updateArticle {
+  fn: import { updateArticle } from "@ext/article/actions.js",
+  entities: [Article]
+}
+
+action deleteArticle {
+  fn: import { deleteArticle } from "@ext/article/actions.js",
+  entities: [Article]
+}
+
+action setArticleFavorited {
+  fn: import { setArticleFavorited } from "@ext/article/actions.js",
+  entities: [Article]
+}
+
+action createComment {
+  fn: import { createComment } from "@ext/article/actions.js",
+  entities: [Comment]
+}
+
+action deleteComment {
+  fn: import { deleteComment } from "@ext/article/actions.js",
+  entities: [Comment]
+}
+
+query getTags {
+  fn: import { getTags } from "@ext/article/queries.js",
+  entities: [ArticleTag, Article]
+}
```

Comment:
huh long file. Maybe put some comments as dividers, e.g. `//--------------- Article ---------------- //`. But better solution would of course be to support multiple files, this is just in the meanwhile so it is easier to read.

Reply:
All right, I organized it a little bit as you suggested!